api/withdrawal.go

```diff
@@ -72,7 +81,10 @@ func (c *Call) FiatWithdrawal(ctx context.Context, request model.WithdrawalReque
 		SetAuthToken(c.bearerToken).
 		SetBody(request).
 		SetResult(&response).
-		SetHeader("Signature", signature).
+		SetHeaders(map[string]string{
+			"Signature":              signature,
```

Comment:
what is Signature?

Reply:
It is value set on the response header, more like authorization, using the public key and the reference passed in each request.

I observed it was not used across board. To remove?